vendor/wp-now/src/wp-now.ts

```diff
@@ -109,6 +113,11 @@ export default async function startWPNow(
 		await activatePluginOrTheme( php, options );
 	}
 
+	// Setup internal plugins needed for Playground
+	await setupPlatformLevelMuPlugins( php );
```

Comment:
This function is called when booting WordPress in Playground: https://github.com/WordPress/wordpress-playground/blob/a55dcc01dcb7ac62b6ea7b99949e01b01f4e92cd/packages/playground/wordpress/src/boot.ts#L138